README.md

```diff
@@ -10,7 +10,11 @@ A NBA conference website (National Basketball Association).
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://arturgouveia1970.github.io/Capstone-NBA/)
+
+## Promocional video
+
+[video link](https://www.loom.com/share/4664af13a22440fcb64bcc145cf01071)
```

Comment:
kindly make your video accessible by anyone. i can not see the video because it private